sdk/storage/Azure.ResourceManager.Storage/src/autorest.md

```diff
@@ -94,12 +167,35 @@ directive:
               $.properties[key] = property;
           }
       }
+  # assigning formats
   - from: swagger-document
-    where: $.definitions
-    transform: >
-      $.StorageAccountCheckNameAvailabilityParameters["x-ms-client-name"] = "StorageAccountNameAvailabilityContent";
-      $.StorageAccountCheckNameAvailabilityParameters.properties.type["x-ms-format"] = "resource-type";
+    where: $.definitions.StorageAccountCheckNameAvailabilityParameters.properties.type
+    transform: $["x-ms-format"] = "resource-type";
+  - from: swagger-document
+    where: $.definitions.DeletedAccountProperties.properties.storageAccountResourceId
+    transform: $["x-ms-format"] = "arm-id";
+  - from: swagger-document
+    where: $.definitions.DeletedAccountProperties.properties.creationTime
+    transform: $["format"] = "date-time";
+  - from: swagger-document
+    where: $.definitions.DeletedAccountProperties.properties.deletionTime
+    transform: $["format"] = "date-time";
+  - from: swagger-document
+    where: $.definitions.StorageAccountProperties.properties.primaryLocation
```

Comment:
Use `format-by-name-rules`, '*location'

Reply:
I would not like to do that since I do not have solid confirmation that others are also locations